common/js_handle.go

```diff
@@ -101,14 +101,10 @@ func (h *BaseJSHandle) Evaluate(pageFunc goja.Value, args ...goja.Value) any {
 }
 
 // EvaluateHandle will evaluate provided page function within an execution context.
-func (h *BaseJSHandle) EvaluateHandle(pageFunc goja.Value, args ...goja.Value) api.JSHandle {
+func (h *BaseJSHandle) EvaluateHandle(pageFunc goja.Value, args ...goja.Value) (api.JSHandle, error) {
 	rt := h.execCtx.vu.Runtime()
 	args = append([]goja.Value{rt.ToValue(h)}, args...)
-	res, err := h.execCtx.EvalHandle(h.ctx, pageFunc, args...)
-	if err != nil {
-		k6ext.Panic(h.ctx, "%w", err)
-	}
-	return res
+	return h.execCtx.EvalHandle(h.ctx, pageFunc, args...)
```

Comment:
Same here, I think we should wrap the error with more context here.

Reply:
`BaseJSHandle` is an interim method. I don't know what we should add here to enhance the returned error. Would you happen to have a suggestion?

Reply:
Ah, ok, that's fine in that case. I wouldn't worry about changing anything here. It was minor anyway 🙂